doc/pod2htmd.tmp

```diff
@@ -0,0 +1,2 @@
+
+.
```

Comment:
@charleszheng44 Please stop using 'git add .' !
If you must, use 'git commit -a'.